src/components/Skeleton/Skeleton.tsx

```diff
@@ -0,0 +1,68 @@
+import React from "react";
+import styled, { keyframes } from "styled-components";
+import { SkeletonProps } from "./types";
+
+const waves = keyframes`
+   from {
+        left: -150px;
+    }
+    to   {
+        left: 100%;
+    }
+`;
+
+const pulse = keyframes`
+  0% {
+    opacity: 1;
+  }
+  50% {
+    opacity: 0.4;
+  }
+  100% {
+    opacity: 1;
+  }
+`;
+
+const Root = styled.div<{ variant: string; width: number | undefined; height: number | undefined }>`
+  min-height: 20px;
+  display: block;
+  background-color: #bdc2c4;
```

Comment:
How this render on dark mode ? Maybe you should use `textDisabled` from the theme. To be confirmed.

Reply:
It is not looking so nice, I am going to see the behaviour for dark mode
![2020-12-21 18 16 19](https://user-images.githubusercontent.com/14844155/102803554-b387d880-43b8-11eb-9671-aaaa78d7598c.gif)

Reply:
Definitely looks much better: 
![2020-12-21 18 18 56](https://user-images.githubusercontent.com/14844155/102803771-14171580-43b9-11eb-98fb-c50e6df345f6.gif)